src/services/bundle-descriptor-converter-service.ts

```diff
@@ -28,17 +28,33 @@ const WIDGET_DESCRIPTOR_VERSION = 'v2'
 const BUNDLE_DESCRIPTOR_VERSION = 'v2'
 const BUNDLE_DESCRIPTOR_NAME = 'descriptor' + DESCRIPTOR_EXTENSION
 
+export enum ThumbnailStatusMessage {
+  NONE = '',
+  FILESIZE_EXCEED = 'Thumbnail filesize is too big. Make sure the thumbnail filesize is 100 KB or below.',
+  OK = 'passed',
+  NO_THUMBNAIL = 'No thumbnail found. Please provide a thumbnail in PNG or JPEG format (e.g. `thumbnail.png`) and place it under the root of your bundle directory'
```

Comment:
No bundle thumbnail file found. Please provide a thumbnail.[png | jpg | jpeg] and place it under the root of your bundle directory

Reply:
done